app/components/FeedPost.module.css

```
@@ -75,14 +78,14 @@
 
 }
 
-@media (--sm) {
+@media (--tablet) {
```

Biome (2.1.2)

[error] Don't use unknown media feature names.

Unexpected unknown media feature name.
You should use media feature names defined in the CSS Specifications.

(lint/correctness/noUnknownMediaFeatureName)